packages/cxl-ui/scss/cxl-catalog.scss

```diff
@@ -0,0 +1,20 @@
+@import 'sass-mq';
+
+:host {
+  .entry__content {
+    border: calc( 0.5 * var(--lumo-margin) ) solid var(--lumo-contrast-30pct);
+    border-radius: var(--lumo-border-radius-s);
+    display: grid;
+    grid-gap: calc( 2 * var(--lumo-margin) );
+    grid-template-columns: repeat( auto-fit, minmax( var(--lumo-data-grid-column-size), 1fr ) );
+    padding: var(--lumo-padding);
```

Comment:
It feels like `grid-gap` and `padding` on grid outer edges should be equal?

Narrow edge space and wide grid gap looks weird.